src/test/java/ikysil/training/ws/api/v1/OrdersResourceTest.java

```diff
@@ -24,27 +37,90 @@
 @RunWith(SpringRunner.class)
 public class OrdersResourceTest {
 
+    OrderDao dao = new InMemoryOrderDao();
+
     public static final String BASE_URL = "/ws/api/v1/orders/";
 
     @LocalServerPort
     private int port;
 
+    private OrderDto order1 = new OrderDto("1144", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("3257", "twenty two", "none", "4024", "seconduser@gmail.com");
+    private OrderDto newOrder = new OrderDto("3257","thirty three", "noneone", "2440", "newuser@gmail.com");
+
     @Before
     public void setUp() {
         RestAssured.basePath = BASE_URL;
         RestAssured.port = port;
     }
 
+    @Before
+    public void savingOK() {
+        given().contentType(ContentType.JSON)
```

Comment:
you do post this two orders in beginning of each test, which I think is overkill(what about getting non existent order from empty DB?) Same with removing orders in @After
@Before and @After should contain some generic logic that is common for all tests.

I propose to rework it as follows:
1 you can fill DB with orders before test directly in dao; for this you need to inject dao:
```
@Autowired
OrderDao dao;
```
and then 
```
dao.saveOrder(order1);
dao.saveOrder(order2);
```

2. have one @Before method, having more than one @Before is confusing

3.  you could add method `OrderDao.removeAll()` to use in @After